javascript/javascript1/week1/index.js

```diff
@@ -23,3 +23,43 @@ if (shouldShowResultInDogYears) {
 else {
     console.log('Your dog will be ' + dogYear + ' human years old in ' + dogYearFuture);
 }
+
+// Housey pricey (A house price estimator)
+
+
+// Peter Hous
+
+const petersWide = 8;
+const petersDeep = 10;
+const petersHigh = 10;
+const peterGardenSizeInM2 = 100,
```

Comment:
Try removing this comma from here, since it is unnecessary and it's causing a syntax error.